packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -51,6 +51,11 @@ const deprecateComponent = deprecate(
 	{name: 'component', replacedBy: 'itemRenderer', until: '2.0.0'}
 );
 
+const deprecateData = deprecate(
+	() => {},
+	{name: 'data', until: '2.0.0'}
```

Comment:
It's not necessary to pass an empty method to deprecate.  Not sure how I missed this before but if you pass a single argument it will print the message immediately. Can do same with `deprecateComponent` as well.

Reply:
In https://github.com/enactjs/enact/pull/1501/commits/318af46e4d35579c09c136f7adb536056c9e4c82, @ryanjduffy tried to print the message once. According to the definition of `deprecate`, we need to pass the function as the first argument and the configuration as the second argument which could not be omitted to do it once. If I missed something, please let me know.